extensions/amp-story/1.0/amp-story-draggable-drawer-header.css

```diff
@@ -81,6 +81,7 @@
   text-overflow: ellipsis !important;
   top: 35px !important;
   white-space: nowrap !important;
+  width: calc(100% - 112px) !important;
```

Comment:
Is this needed for this PR?

Reply:
This is from another commit, not clear why it is showing up in this PR

Reply:
Looks like this was resolved.